server/testutil/testenv/testenv.go

```diff
@@ -124,29 +122,11 @@ func writeTmpConfigFile(testRootDir string) (string, error) {
 	return configPath, nil
 }
 
-func ensureHomeDir(t testing.TB) {
```

Comment:
@tempoz FYI I had added this code in a previous PR because I mistakenly thought a test configuration change I had made was causing this error with HOME / XDG_CONFIG_HOME not being defined. Turns out the error was something else, so this can be reverted.